instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/rpc/RpcMetricsAdvice.java

```diff
@@ -45,7 +48,45 @@ static void applyServerDurationAdvice(DoubleHistogramBuilder builder) {
     // https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-metrics.md
     ((ExtendedDoubleHistogramBuilder) builder)
         .setAttributesAdvice(
-            Arrays.asList(
+            asList(
+                RpcCommonAttributesExtractor.RPC_SYSTEM,
+                RpcCommonAttributesExtractor.RPC_SERVICE,
+                RpcCommonAttributesExtractor.RPC_METHOD,
+                RPC_GRPC_STATUS_CODE,
+                NetworkAttributes.NETWORK_TYPE,
+                NetworkAttributes.NETWORK_TRANSPORT,
+                ServerAttributes.SERVER_ADDRESS,
+                ServerAttributes.SERVER_PORT));
+  }
+
+  static void applyClientRequestSizeAdvice(LongHistogramBuilder builder) {
```

Comment:
is you are always going to use the same set of attributes you could make all these delegate to a common method or extract the attributes list to a variable

Reply:
Done.